server/src/gql/resolver.py

```diff
@@ -0,0 +1,60 @@
+"""GraphQL Ariadne resolver functions."""
+# 'annotations' import needs to return enclosing type:
+# https://www.python.org/dev/peps/pep-0563/
+from __future__ import annotations
+
+from typing import Any, List
+
+from ariadne import MutationType, QueryType, ScalarType
+from ariadne.types import GraphQLResolveInfo  # type: ignore
+from bson.objectid import ObjectId
+from flask_login import current_user
+from src.gql.graphql import NewMatesInput
+from src.model.family import Family
+from src.model.mate import Mate
+from src.model.user import User
+
+oid_scalar = ScalarType("oid")
+
+
+@oid_scalar.serializer
+def serialize_oid(id: ObjectId) -> str:
+    """
+    Convert all MongoDB ObjectID into str.
+
+    Otherwise, objects cannot be returned via JSON serialization.
+    Returns str form of the id
+    """
+    return str(id)
+
+
+query = QueryType()
+mutation = MutationType()
+
+
+@mutation.field("addNewMates")
+def resolve_add_new_mates(
+    obj: Any, info: GraphQLResolveInfo, mates: List[NewMatesInput]
+) -> List[Mate]:
+    """Add new mates to the user's unassigned family."""
+    user = User.lookup_user(current_user.get_id())
+    if not user:
+        raise Exception("User not found")
+
+    family = Family.lookup_family(user.unassigned_family_id)
+    if not family:
+        family_error, family = Family.add_new_family("Unassigned", 365)
```

Comment:
what's significance of 365? And can we make it a constant somewhere.

Reply:
good idea